### PR TITLE
examples: remove ipfs components from echo

### PR DESCRIPTION
**Pull request by ZuoFuhong**

### CODE REVIEW

File: examples/echo/main.go [hunk 1]
```diff
@@ -18,19 +18,13 @@ import (
...
 )
 
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	// LibP2P code uses golog to log messages. They log with different
-	// string IDs (i.e. "swarm"). We can control the verbosity level for
-	// all loggers with:
-	golog.SetAllLoggers(golog.LevelInfo) // Change to INFO for extra info
```
Comment (marten-seemann) [inline examples/echo/main.go:32]: This is useful, please don't remove.

**Review by marten-seemann — COMMENTED** [review]
This looks like a worthwhile change, but please don't remove the logging part.

File: examples/echo/main.go (hunk 1 above)
Comment (ZuoFuhong) [reply examples/echo/main.go:32]: I think it is possible to use the standard library log.

Comment (Stebalien) [reply examples/echo/main.go:32]: We use go-log extensively in libp2p, so this doesn't really help.

Comment (ZuoFuhong) [reply examples/echo/main.go]: Ok，i undo

**Review by marten-seemann — APPROVED** [review]
LGTM, thank you!

**Pull request merged**